packages/botonic-cli/src/utils.ts

```diff
@@ -80,7 +84,16 @@ function getSystemInformation() {
     arch: os.arch(),
     timezone: Intl.DateTimeFormat().resolvedOptions().timeZone,
     timestamp: new Date().toISOString(),
-    botonic_cli_version: getBotonicCliVersion(),
+    is_tty: process.stdout.isTTY,
+    binary_path: isWindows()
+      ? execCommand('where botonic')
+      : execCommand('which botonic'),
```

Comment:
The purpose of `binary_path` is to learn where the framework is installed in case there's the install problem and `botonic` command is not found. So with this approach it will fail.
I propose to just get the current directory with the global variable `__dirname` (https://stackoverflow.com/questions/3133243/how-do-i-get-the-path-to-the-current-script-with-node-js)

Reply:
done @ericmarcos ! 👍